### ISSUE TITLE
Ensure site meets Accessibility Standards

### ISSUE THREAD

**Issue by joebeernink** [TrashMob-eco/TrashMob#197]
Review all site for accessibility issues and address.

Is there a tool that can assist with this?

**Comment by joebeernink** [comment]
This is part of the redesign effort, and the accessibility tool should be run frequently to check for issues
